src/models/tools/image/image-import-export.ts

```diff
@@ -1,41 +1,50 @@
-import { createChange, ImageToolChange } from "./image-change";
-import { ITileExportOptions } from "../tool-content-info";
 import { safeJsonParse } from "../../../utilities/js-utils";
+import { ITileExportOptions } from "../tool-content-info";
 
-export interface IImageTileImportSpec {
+export interface ILegacyImageTileImport {
   type: "Image";
   url: string;
+  changes: string[];
 }
 
-export const isImageTileImportSpec = (snapshot: any): snapshot is IImageTileImportSpec =>
-              (snapshot?.type === "Image") && (snapshot.url != null) && !snapshot.changes;
+export const isLegacyImageTileImport = (snapshot: any): snapshot is ILegacyImageTileImport => {
+  return (snapshot?.type === "Image") && !!snapshot.changes;
+};
+
+export const convertLegacyImageTile = (snapshot: ILegacyImageTileImport) => {
+  const { changes, url, ...others } = snapshot;
+  let changeUrl = "", changeFilename = "";
+  if (snapshot.changes.length > 0) {
+    const changeObj = safeJsonParse(snapshot.changes[changes.length-1]);
```

Comment:
One more thing I should have mentioned -- harking back to something @scytacki mentioned in his review, it would be good to maintain type information here rather than resorting to `any` as this code is currently doing, e.g.:
```suggestion
    const changeObj = safeJsonParse<ImageToolChange>(snapshot.changes[changes.length-1]);
```
which will require keeping the definition of `ImageToolChange` around, but it can be defined locally in this file since apparently no other code references it.